src/main/java/digital/slovensko/autogram/ui/gui/SettingsDialogController.java

```diff
@@ -0,0 +1,212 @@
+            userSettings.setDriver(newValue);
+        });
+    }
+
+    private void initializeEn319132CheckBox() {
+        en319132CheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            userSettings.setEn319132(newValue);
+        });
+        en319132CheckBox.setSelected(userSettings.isEn319132());
+    }
+
+    private void initializeCorrectDocumentDisplayCheckBox() {
+        correctDocumentDisplayCheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            userSettings.setCorrectDocumentDisplay(newValue);
+            correctDocumentDisplayCheckBox.setText(newValue ? "Zapnutá" : "Vypnutá");
+        });
+        correctDocumentDisplayCheckBox.setSelected(userSettings.isCorrectDocumentDisplay());
+    }
+
+    private void initializeSignatureValidationCheckBox() {
+        signatureValidationCheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            userSettings.setSignaturesValidity(newValue);
+            signatureValidationCheckBox.setText(newValue ? "Zapnutá" : "Vypnutá");
+        });
+        signatureValidationCheckBox.setSelected(userSettings.isSignaturesValidity());
+    }
+
+    private void initializeCheckPDFAComplianceCheckBox() {
+        checkPDFAComplianceCheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            userSettings.setPdfaCompliance(newValue);
+            checkPDFAComplianceCheckBox.setText(newValue ? "Zapnutá" : "Vypnutá");
+        });
+        checkPDFAComplianceCheckBox.setSelected(userSettings.isPdfaCompliance());
+    }
+
+    private void initializeLocalServerEnabledCheckBox() {
+        localServerEnabledCheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            userSettings.setServerEnabled(newValue);
+            localServerEnabledCheckBox.setText(newValue ? "Zapnutý" : "Vypnutý");
+        });
+        localServerEnabledCheckBox.setSelected(userSettings.isServerEnabled());
+    }
+
+    private void initializeBatchSigningRadioButtons() {
+        toggleGroup = new ToggleGroup();
+
+        var togetherRadioButton = new RadioButton("Spoločne");
+        togetherRadioButton.setToggleGroup(toggleGroup);
+        batchSigningRadioButtons.getChildren().add(togetherRadioButton);
+
+        var individaullyRadioButton = new RadioButton("Samostatne");
+        individaullyRadioButton.setToggleGroup(toggleGroup);
+        batchSigningRadioButtons.getChildren().add(individaullyRadioButton);
+
+        individaullyRadioButton.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            userSettings.setSignIndividually(newValue);
+        });
+
+        if (userSettings.isSignIndividually()) {
+            individaullyRadioButton.setSelected(true);
+        } else {
+            togetherRadioButton.setSelected(true);
+        }
+    }
+
+    private void initializeTrustedCountriesList() {
+        var europeanCountries = List.of(
```

Comment:
Tento list by tiež bolo fajn prekladať na skratky: SK, CZ, ... Tak sa potom používajú pri overovaní. Ale ak je to zložité, dorobím to potom do validácie.